committee.html

```diff
@@ -437,21 +437,21 @@ <h1>THE COMMITTEE</h1>
                 <div class="card shadow-sm h-100">
                   <svg class="bd-placeholder-img card-img-top" viewBox="0 0 100 100" role="img"
                     preserveAspectRatio="xMidYMid meet" focusable="false">
-                    <image href="img/team/michael.jpeg" width="100%" height="100%">
+                    <image href="img/team/maddi.jpeg" width="100%" height="100%">
                   </svg>
                   <div class="card-body">
                     <p class="card-text custom-font bold-font darkcolor"
                       style="font-size:160%; margin-bottom: 0; padding-top: 10px !important;">
-                      Michael de Campos</p>
+                      Maddi Wong</p>
```

Comment:
For these containers, make sure they have unique identifiers than other ones on the page (remember Maddi Wong still exists somewhere else in the code for someone else to change), so perhaps just spell maddi here wrong, maybe capitalize it in ways that makes it distinct, make it distinct from the maddi containers that are old and need to be changed. Same thing with Vivian's old and new bios.